meta-mender-core/recipes-update-modules/mender-app-update-module/mender-app-update-module_git.bb

```diff
@@ -0,0 +1,53 @@
+DESCRIPTION = "Mender Application Update Module for supporting containerized application updates on devices including related tools."
+
+HOMEPAGE = "https://mender.io"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=b4b4cfdaea6d61aa5793b92efd42e081"
+
+SRC_URI = "git://github.com/mendersoftware/app-update-module;branch=master;protocol=https"
+
+SRCREV = "15025d9f1e21eafcbf1325b2a5034b2606e1d719"
+
+S = "${WORKDIR}/git"
+
+RDEPENDS:${PN} = "jq mender-client xdelta3"
+
+PACKAGECONFIG ?= ""
+PACKAGECONFIG[docker-compose] = ",,,docker-compose"
```

Comment:
Shouldn't `k8s` be a second packageconfig then? Additionally, a check could be added to issue a warning if neither is set - because then the module won't do anything in the end.

Reply:
Yes, can be. What shall `k8s` depend on? `k3s`? How can I depend on something virtual as "kubernetes"?

Reply:
Added `k3s` and `k8s`.